hikari/impl/rest.py

```diff
@@ -2137,9 +2138,98 @@ async def delete_emoji(
         self,
         guild: snowflakes.SnowflakeishOr[guilds.PartialGuild],
         emoji: snowflakes.SnowflakeishOr[emojis.CustomEmoji],
+        *,
+        reason: undefined.UndefinedOr[str] = undefined.UNDEFINED,
     ) -> None:
         route = routes.DELETE_GUILD_EMOJI.compile(guild=guild, emoji=emoji)
-        await self._request(route)
+        await self._request(route, reason=reason)
+
+    async def fetch_available_sticker_packs(self) -> typing.Sequence[stickers.StickerPack]:
+        route = routes.GET_STICKER_PACKS.compile()
+        response = await self._request(route)
+        assert isinstance(response, dict)
+        return [self._entity_factory.deserialize_sticker_pack(pack) for pack in response["sticker_packs"]]
+
+    async def fetch_standard_sticker(
```

Comment:
This endpoint returns both sticker types though, it just never returns the "user" field for GuildStickers since that's privileged info.